P/PALISADE/build_tarballs.jl

```diff
@@ -31,7 +31,7 @@ make installcore installpke installabe installsignature "${FLAGS[@]}"
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
 platforms = [
-    Linux(:x86_64; compiler_abi=CompilerABI(:gcc6)),
+    Platform("x86_64; compiler_abi=CompilerABI(:gcc6", "linux")),
     Linux(:x86_64; libc=:musl, compiler_abi=CompilerABI(:gcc6)),
```

Comment:
```suggestion
    Platform("x86_64", "linux"; libgfortran_version="3.0.0"),
    Platform("x86_64", "linux"; libc="musl", libgfortran_version="3.0.0"),
```

Reply:
Maybe this needs to set `preferred_gcc_version=v"6"` rather restricting the libgfortran version

Reply:
It was a mistake (due to a limitation of my script) to change this line at all -- I'd prefer to actually undo this change here, and leave it to a subsequent cleanup to clean this up. Yeah, it sounds like `preferred_gcc_version=v"6"` is more appropriate, but I'd rather not be the one to dig in and find out and/or risk breaking this JLL about which I know nothing.